eos/saveddata/fit.py

```diff
@@ -461,11 +461,16 @@ def calculateModifiedAttributes(self, targetFit=None, withBoosters=False, dirtyS
         timer = Timer(u'Fit: {}, {}'.format(self.ID, self.name), logger)
         logger.debug("Starting fit calculation on: %r, withBoosters: %s", self, withBoosters)
 
+        try:
+            projectionInfo = self.getProjectionInfo(targetFit.ID)
+        except:
+            pass
+        else:
+            logger.debug("ProjectionInfo: %s", projectionInfo)
+
```

Comment:
> pyCharm complains that it's assigned and not used, because it knows it's a bullshit assignment. ☺

=/

yeah, revert it for now. Also, I'm not sure which pyCharm warning you're getting since I turned most warnings off as it pertains to things like this. >_>